packages/document-grounding/README.md

```diff
@@ -92,6 +92,9 @@ const response: DocumentsListResponse = await VectorApi.createDocuments(
 ).execute();
 ```
 
+**Note:** Previous versions of this documentation referenced the `text-embedding-ada-002` model, which is now deprecated and scheduled for retirement on 2025-10-03.
+All examples have been updated to use the recommended replacement models: `text-embedding-3-small` or `text-embedding-3-large`.
```

Comment:
[pp] Can we centralize this into a section in the main README.md? Something like `Deprecated Model` and say stop using this and that, replace them with xyz. This would be enough IMO.